src/cogs/trivia.py

```diff
@@ -170,14 +170,29 @@ def check(message) -> bool:
     async def top(self, ctx):
         """Sends an embed with the top 5 ranked users in trivia"""
         embed = nextcord.Embed(title="Trivia Leaderboard")
+        last = ()  # Declare a tuple in function scope to hold the amount of wins and losses of the last person checked
         i = 1
         for leader in self.cur.execute(
                 'SELECT id, rank, losses FROM score ORDER BY rank DESC LIMIT 5'
         ):
-            embed.add_field(name=f"{i}. {self.bot.get_user(leader[0]).name}",
-                            value=f"{leader[1]} wins, {leader[2]} losses",
-                            inline=False)
+            if i != 1:  # As long as it's not the first person to be checked continue
+                if (leader[1], leader[2]) == last:  # If the wins and losses are equal to the last person check they should be ranked equally
+                    embed.add_field(name=f"{i-1}. {self.bot.get_user(leader[0]).name}",
```

Comment:
When >2 people are tired for the same position, this happens:
![image](https://user-images.githubusercontent.com/15039983/158028160-9c5d71c4-f655-481d-bce4-5209fe4a8417.png)